doc/developer-guide/api/functions/TSHttpTxnServerReqGet.en.rst

```diff
@@ -26,7 +26,28 @@ Synopsis
 
 `#include <ts/ts.h>`
 
-.. function:: TSReturnCode TSHttpTxnServerReqGet(TSHttpTxn txnp, TSMBuffer * bufp, TSMLoc * offset)
+.. function:: TSReturnCode TSHttpTxnServerReqGet(TSHttpTxn txnp, TSMBuffer * bufp, TSMLoc * obj)
 
 Description
 ===========
+
+Get the request |TS| is sending to the upstream (server) for the transaction :arg:`txnp`.
+:arg:`bufp` and :arg:`obj` should be valid pointers to use as return values. The call site could
+look something like ::
+
+   TSMBuffer mbuffer;
+   TSMLoc mloc;
+   if (TS_SUCCESS == TSHttpTxnServerReqGet(&mbuffer, &mloc)) {
```

Comment:
The API is TSHttpTxnServerReqGet(txn, &mbuffer, &mloc), it takes 3 arguments.
Can you describe what are mbuffer and mloc? It looks like mbuffer points to the request while mloc points to the header. In case it's obvious or assumed to be a prerequisite never mind.

Reply:
See [here - TSMBuffer](https://docs.trafficserver.apache.org/en/latest/developer-guide/api/functions/TSTypes.en.html#c.TSMBuffer) and [here - TSMLoc](https://docs.trafficserver.apache.org/en/latest/developer-guide/api/functions/TSTypes.en.html#c.TSMLoc). When the docs are generated, the types in the function signature will be linked to those definitions.